src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -409,8 +409,9 @@
     </PropertyGroup>
     <ItemGroup>
       <!-- Exclude F# from retargeting: https://github.com/dotnet/toolset/issues/2866 -->
```

Comment:
this issue was fixed in runtime in year 2019

Reply:
@kasperk81 thx for letting me know. I don't want to mix removing the current exclusions into this PR.

@vzarytovskii Fsharp tooling is excluded from retargetting (the task does text replace to change the target framework to match the version that the SDK is shipping with). According to the comment above, there is a fix in place since 2019. Please have a look into removing this exclusion.